packages/ember-runtime/tests/mixins/array_test.js

```diff
@@ -24,8 +24,8 @@ import { A as emberA } from '../../system/native_array';
 const TestArray = EmberObject.extend(EmberArray, {
   _content: null,
 
-  init(ary = []) {
-    this._content = ary;
+  init() {
```

Comment:
that content array stuff is whack. I should take another look at it I can't remember why it even was like that.